handlebars/src/main/java/com/github/jknack/handlebars/Handlebars.java

```diff
@@ -740,6 +746,39 @@ public boolean stringParams() {
     return stringParams;
   }
 
+
+  /**
+   * If true, given partial blocks are not evaluated when defined but when used.
```

Comment:
Let's add two examples when this flag is on/off

Reply:
Hey I added two tests that deal with this pull request's changes. Is everything okay now?